tests/Jenkins/utilities.sh

```diff
@@ -795,7 +795,7 @@ diracServices(){
     echo "==> calling dirac-install-component $serv ${DEBUG}"
     if ! dirac-install-component "$serv" "${DEBUG}"; then
       echo 'ERROR: dirac-install-component failed' >&2
-      exit 1
+      #exit 1
```

Comment:
Why?

Reply:
It's artefacts from testing, I'll fix it, thnx